Formula/r/rtx.rb

```diff
@@ -24,9 +24,11 @@ class Rtx < Formula
   end
 
   def install
-    system "cargo", "install", "--features=brew", *std_cargo_args
```

Comment:
> but now it's removed so it will fail if passed I think

can you elaborate a bit? I dont think I understand fully

(also looks. like the build indeed passed?)

Reply:
I think @jdx is saying that the `brew` feature was [removed in 2023.12.](https://github.com/jdx/rtx/compare/v2023.12.5...v2023.12.7#diff-2e9d962a08321605940b5a657135052fbcef87b5e360662bb527c96d9a615542)6 and so keeping the line around causes it to fail.